simulations.py

```diff
@@ -0,0 +1,247 @@
+    longitude: static location longitude in decimal degrees
+  """
+
+  def __init__(self, run_time, gain, latitude, longitude):
+    Simulation.__init__(self, run_time, gain)
+    self.latitude = latitude
+    self.longitude = longitude
+    coordinate = "%s,%s" % (latitude, longitude)
+    self.process_args = ["./run_bladerfGPS.sh", "-l", coordinate, "-T", "now",
+                         "-d", str(self.run_time), "-a", str(self.gain)]
+
+  def __repr__(self):
+    return "StaticSimulation(run_time=%s, gain=%s, latitude=%s, longitude=%s)" % (self.run_time, self.gain, self.latitude, self.longitude)
+
+
+class DynamicSimulation(Simulation):
+  """An object for a single GPS Simulation for a static location.
+
+  Attributes:
+    run_time: simulation duration in seconds
+    gain: signal gain for the broadcast by bladeRF board
+    process: python subprocess created for the simulation
+    process_args: arguments for command to pass into subprocess to run the
+    simulation in bladeGPS
+    start_time: DateTime object for most recent start time
+    end_time: DateTime object for most recent end time
+    file_name: name user motion csv file for dynamic route simulation
+    to be used from the user_motion_files directory
+  """
+
+  def __init__(self, run_time, gain, file_name):
+    Simulation.__init__(self, run_time, gain)
+    self.file_name = file_name
+    file_path = "../geobeam/user_motion_files/" + file_name
+    self.process_args = ["./run_bladerfGPS.sh", "-u", file_path, "-T", "now",
+                         "-d", str(self.run_time), "-a", str(self.gain)]
+
+  def __repr__(self):
+    return "DynamicSimulation(run_time=%s, gain=%s, file_name=%s)" % (self.run_time, self.gain, self.file_name)
+
+
+class SimulationSet():
+  """An object for a set of GPS simulations (that can be dynamic or static).
+
+  Attributes:
+    simulations: list of simulation objects in order of desired execution
+    current_simulation: simulation object currently being run
+    current_simulation_index: index of currently run simulation object in list
+    log_filename: string for name of file to log into simulation_logs folder
+  """
+
+  def __init__(self, simulations):
+    self.simulations = simulations
+    self.current_simulation = None
+    self.current_simulation_index = None
+    now = datetime.datetime.utcnow()
+    self.log_filename = now.strftime("%Y-%m-%d_%H:%M:%S")
+
+  def run_simulations(self):
+    """Starts simulations and navigates through according to user key press.
+
+    Starts the first simulation, and then continuously checks if the current
+    simulation is running, and switches to the next or previous based on
+    keyboard input. If user presses q or last simulation finishes, it ends
+    the simulation set
+    """
+    self.current_simulation_index = 0
+    self.current_simulation = self.simulations[self.current_simulation_index]
+    self.current_simulation.run_simulation()
```

Comment:
seems like we could probably create a function for this - run_simulation(index) or another name.
line 203, 207, 208 is also the same